packages/data-stores/src/plans/types.ts

```diff
@@ -162,6 +173,15 @@ export interface PricedAPIPlanIntroductoryOffer {
 	introductory_offer_end_date?: string;
 }
 
+export interface PricedAPIPlanCostOverrides {
```

Comment:
I think this should be `PricedAPIPlanCostOverride` (singular)?

Reply:
Nice catch!